frontend/src/components/program/FormScheduleEntryItem.vue

```diff
@@ -19,6 +19,7 @@
           :name="$tc('components.program.formScheduleEntryItem.start')"
           vee-id="startDatetime"
           vee-rules="required"
+          :max="maxStartTime"
```

Comment:
Before, I could shift the start time to any time of the day, which shifted the entry and the duration.

Now this is not possible anymore: To shift an entry to a later date, I have to first change the end time and only then I can change the start time. But then I have to change the end date again, because the duration was kept. 😅 
This seems akward to me.

Reply:
+1 for a range datetime picker in the future. For now: Good point, I was not aware of any logic we have so far involving auto-shifting the end date with the start date. I can have a look and re-activate that.

Reply:
> I was not aware of any logic we have so far involving auto-shifting the end date with the start date. I can have a look and re-activate that.

The log is still there & enabled:
https://github.com/ecamp/ecamp3/pull/3172/files#diff-0e34dccf17184e2244a9d147e7ef554a40281d2d5431375a80bce90ccd099386R149

Reply:
Fixed in [2ebe629](https://github.com/ecamp/ecamp3/pull/3172/commits/2ebe629ab3e8bac8756b26a7393185fb01f6f496)